src/main/java/org/broadinstitute/hellbender/cmdline/argumentcollections/VariantAnnotationArgumentCollection.java

```diff
@@ -0,0 +1,49 @@
+package org.broadinstitute.hellbender.cmdline.argumentcollections;
+
+import com.google.common.collect.Lists;
+import org.broadinstitute.barclay.argparser.Advanced;
+import org.broadinstitute.barclay.argparser.Argument;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+public class VariantAnnotationArgumentCollection implements Serializable {
+    private static final long serialVersionUID = 1L;
+
+    public VariantAnnotationArgumentCollection(List<String> defaultGroups, List<String> defaultAnnotations, List<String> defaultExclude) {
+        annotationGroupsToUse = new ArrayList<>(defaultGroups);
+        annotationsToUse = new ArrayList<>(defaultAnnotations);
+        annotationsToExclude = new ArrayList<>(defaultExclude);
+    }
+
+    /**
+     * Which annotations to recompute for the combined output VCF file.
+     */
+    @Advanced
```

Comment:
Should this really be marked as `@Advanced`? It doesn't strike me as an advanced argument. What do you think @davidbenjamin ?

Reply:
I agree that this isn't advanced.